app/views/vendors/_sales.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="list">
+  <ul>
+    <% sales.each do |sale| %>
+      <div class="list-item">
+        <li>
+          $<%= number_with_precision(sale.amount, precision: 2) %>, <%= sale.purchase_time %>
```

Comment:
Rails has some handy time formatters to make dates/times easier on the eyes. Check them out! :)